cli/flox/doc/flox-install.md

```diff
@@ -67,7 +74,10 @@ options.
 
 `<package>`
 :   The pkg-path of the package to install as shown by 'flox search'
-    Append `@<version>` to specify a version requirement
+    Append `@<version>` to specify a version requirement.
+
+    Alternatively, an arbitrary Nix flake output attribute may be specified.
```

Comment:
Suggestion: I'm not sure it makes sense to keep referring to this as a "flake output attribute" because that implies that you need to do something more than `github:flox/flox`, which has a default attribute that you'll get without specifying anything else.

Reply:
A flake output attribute doesn't have to have an attrpath per https://nix.dev/manual/nix/2.17/command-ref/new-cli/nix#flake-output-attribute

But that's obviously a bit of a technicality given you misunderstood.

Should we use flake package?

Reply:
I don't like flake reference because that implies there isn't an attribute path

Reply:
I mean I know that a flake output doesn't need an attrpath, that's kind of what I was saying. The term is a little confusing (regardless of whether it's the correct Nix term). Can we just say a "flake installable"?

Reply:
Changed to flake installable

Reply:
Think it's okay to use flake output attribute in manifest.toml.md?

Reply:
Probably better to be consistent and use the same terminology in both places

Reply:
After that LGTM

Reply:
Changed with an added `which Nix refers to as a flake output attribute`